src/llama_stack_provider_kft/kft_adapter.py

```diff
@@ -103,6 +142,72 @@ async def handler(
                 self.config.gpu_identifier: self.config.nproc_per_node,
             }
 
+            init_containers = None
+            logger.info(f"Preprocess: {self.config.preprocess}")
+            if self.config.preprocess:
+                dp_master_args = [
+                    f"""
+                    echo "Processing data"
+                    echo "Using "{self.config.model_path}" model for training"
+                    echo "Using "{self.config.data_path}"  data for training"
+
+                    pip install instructlab-training==0.7.0
```

Comment:
Is it really necessary given the default image that's used for the PyTorchJob?

Reply:
possibly not, but I will say I do know that the training library is changing its data processing, so it might be beneficial to pin to this until we are ready to switch

Reply:
I think we should keep this to be safe, but we can consider loosening in the future